### PR TITLE
Bump packages

### PR DESCRIPTION
**Pull request by ismailToyran**
Bump packages
Remove `useThemeBackground` hook
Change deprecated `useDimensions` chakra hook with `useSize` hook

### CODE REVIEW

**Comment by antho1404** [comment]
Have you tried to install it on the starter kit, for example?
I can't make it work even with your version of https://github.com/liteflow-labs/starter-kit/pull/330 by installing the locally built library.
I have the following errors:
> Module not found: Can't resolve 'bufferutil' in '/Users/antho/prog/chat/node_modules/ws/lib'
> Module not found: Can't resolve 'utf-8-validate' in '/Users/antho/prog/chat/node_modules/ws/lib'

Code looks good, but I can't really test, example app works great tho

File: example/package-lock.json
Comment (antho1404) [inline example/package-lock.json:1]: I have diffs in local when I run `npm i` are you sure this is the latest version?

File: package-lock.json
Comment (antho1404) [inline package-lock.json:1]: I have diffs in local when I run npm i are you sure this is the latest version?

**Comment by ismailToyran** [comment]
Hmm, I've spent quite time on this one, but issue might be related with `XmtpProvider`
Couldn't find a way to fix the issue.
I believe updating the repo might be fix, but looks kinda complicated with our version.
And I believe they've deprecated the version we have been using.
https://github.com/xmtp/xmtp-web/tree/main/packages/react-sdk

image

What do you suggest for this one?
Also for some reason, when I've updated next-translate, there was an error just related with chat page, so had to downgrade that one as well. So there is an issue deeper on this package which I couldn't find.

**Comment by antho1404** [comment]
This error is because of different react versions usually but here I think there is some issue when installing locally with `npm I ./path-to-chat` instead, we can pack the library `npm pack` and then install the package with `npm i ../chat/nft-chat-0.1.12.tgz` and this solves this problem.

I still had a problem because of the proto library that contains a breaking change but was not well-versioned, so right now, I fixed it. I also started to work on an update of the xmtp package https://github.com/liteflow-labs/chat/pull/33, but we can discuss if it's worth updating

**Review by antho1404 — APPROVED** [review]

**Pull request merged**